.ci/script_linux.sh

```diff
@@ -9,10 +9,20 @@ set -e
 ./scripts/internal-run.sh catkin build --no-status --no-deps -p 1 -i --make-args check-format -- aikido
 
 # Manually build Aikido's tests; they are not built automatically because it is not a Catkin package.
-./scripts/internal-run.sh catkin build --no-status --no-deps -p 1 -i --cmake-args -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DTREAT_WARNINGS_AS_ERRORS=ON -DCODECOV=$CODECOV --make-args tests -- aikido
+if [ $BUILD_NAME = TRUSTY_FULL_RELEASE ]; then
```

Comment:
Shouldn't this be `TRUSTY_FULL_DEBUG`? (That's when `CODECOV` used to be `ON`, and I think it might be necessary to run with debugging information?)